.travis.yml

```diff
@@ -1,27 +1,27 @@
-dist: xenial
-sudo: required
+dist: bionic
 language: go
 go:
-  - "1.12.x"
   - "1.13.x"
 
-go_import_path: github.com/containerd/zfs
+env:
+  global:
+    - GOLANGCI_LINT_VERSION=v1.22.2
 
 install:
   - sudo apt-get install -y zfsutils-linux && sudo modprobe zfs
-  - cd $GOPATH/src/github.com/containerd/zfs
-  - GO111MODULE="on" go mod vendor
-  - go get -u github.com/vbatts/git-validation
-  - go get -u github.com/kunalkushwaha/ltag
+  - go mod vendor
+  - pushd ..; go get -u github.com/vbatts/git-validation; popd
+  - pushd ..; go get -u github.com/kunalkushwaha/ltag; popd
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCI_LINT_VERSION}
 
 before_script:
   - pushd ..; git clone https://github.com/containerd/project; popd
 
 script:
   - DCO_VERBOSITY=-q ../project/script/validate/dco
   - ../project/script/validate/fileheader ../project/
-  - go test -race -covermode=atomic -c .
-  - sudo ./zfs.test -test.root -test.v -test.coverprofile=coverage.txt
+  - GOGC=75 golangci-lint run
+  - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
needs sudo

Reply:
same for AUFS: https://github.com/containerd/aufs/pull/18/files#r363200701

Reply:
Got you.